src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceShareService.java

```diff
@@ -117,6 +120,11 @@ public EvidenceShareService(
             }
         }
         log.info("Feature flag turned off for sending to DWP. Skipping evidence share for case id {}", sscsCaseDataCallback.getCaseDetails().getId());
+        if (sscsCaseDataCallback.getCaseDetails().getState().toString().equals(State.VALID_APPEAL.toString())) {
+            log.info("Sending case back to appeal created so it is in correct state for old workflow for case id {}", sscsCaseDataCallback.getCaseDetails().getId());
+            ccdService.updateCase(caseData, Long.valueOf(caseData.getCcdCaseId()), EventType.MOVE_TO_APPEAL_CREATED.getCcdType(), "Case created", "Sending back to appealCreated state", idamService.getIdamTokens());
```

Comment:
Is this  `EventType.MOVE_TO_APPEAL_CREATED` correct?

Reply:
Yes, this is to get the case back into the old workflow whilst we are still migrating to the new workflow